radix-engine/src/updates/cuttlefish.rs

```diff
@@ -466,3 +478,121 @@ fn generate_cuttlefish_update_min_rounds_per_epoch<S: SubstateDatabase + ?Sized>
         consensus_manager_config.into_locked_substate(),
     )
 }
+
+/// A quick macro for encoding and unwrapping.
+macro_rules! scrypto_encode {
+    (
+        $expr: expr
+    ) => {
+        ::radix_common::prelude::scrypto_encode($expr).unwrap()
+    };
+}
+
+fn generate_cuttlefish_update_identity_to_not_create_royalty_module<
+    S: SubstateDatabase + ?Sized,
+>(
+    db: &S,
+) -> StateUpdates {
+    let reader = SystemDatabaseReader::new(db);
```

Comment:
Do we also want to consider updating all the TYPE_INFO substates for _existing_ Identities to disable their royalty modules? (assuming it wouldn't break Gateway)

Reply:
I thought about this - to remove all the substates as a result of the royalty module. 

I felt it's too big a change and it's safer to do it post-cuttlefish (when we have a known list of identities with royalty module).

Reply:
Makes sense to me